github3/events.py

```diff
@@ -30,6 +33,9 @@ class Event(GitHubCore):
     """
 
     def _update_attributes(self, event):
+        # The _json_data is modified here
+        self._orig_json_data = copy.deepcopy(event)
```

Comment:
The thing I was worried about with that is that `__getattr__` uses `_json_data` for its lookup table. I think that would mean that the payload attributes when fetched via `event.pull_request` would then return the dict rather than the pull request object. I'll test that out though.